src/main/java/com/spider/solomon/order/CustomerOrderDao.java

```diff
@@ -34,7 +36,13 @@ public CustomerOrder getOrderById (int orderId) {
 	 */
 	public List<CustomerOrder> getOrdersByCustomerId (int customerId) {
 		String sql = "SELECT * FROM customer_orders WHERE customer_id = ?";
-		return jdbcTemplate.query(sql, ORDER_MAPPER, customerId);
+
+		List<CustomerOrder> ordrs = jdbcTemplate.query(sql, ORDER_MAPPER, customerId);
```

Comment:
Could we rename this to `orders` for clarity and consistency?

[Meta: This is a nitpick comment that I would never block a merge for. If I'm giving other feedback on a PR, I might as well include nitpicks too. I'm ok if the author ignores naming suggestions.]